pypeit/scripts/setup_coadd2d.py

```diff
@@ -18,8 +18,9 @@ def get_parser(cls, width=None):
         input_group = parser.add_mutually_exclusive_group(required=True)
         input_group.add_argument('-f', '--pypeit_file', type=str, default=None,
                                  help='PypeIt reduction file')
-        input_group.add_argument('-d', '--science_dir', type=str, default=None,
-                                 help='Directory with spec2d files to stack')
+        input_group.add_argument('-d', '--science_dir', type=str, nargs='*', default=None,
```

Comment:
I think I've had a similar comment before, but can we change nargs to '+' instead of '*'?  '+' requires at least one argument.